### ISSUE TITLE
what if I don't set episode_done:True?

### ISSUE THREAD

**Issue by andysun-sw** [facebookresearch/ParlAI#4397]
I want to finetune reddit model with question and answer pairs with following command. we know that we can set episode_done:True at the end of  the dialogue, but do I need set episode_done:True every q&a pairs? If I don't set why system remind me with "The data in ... looks very long episode. If this is intentional,you may ignore this" . What's the real purpose of the episode_done:True? does this mean we   put all episode dialogue in the input context when training gpt kind model?

parlai train_model -t amazon_qa   -veps 1 --attention-dropout 0.0 --batchsize 16 --model transformer/generator --embedding-size 2560 --ffn-size 10240 --variant prelayernorm --n-heads 32 --n-positions 128 --n-encoder-layers 2 --n-decoder-layers 24 --history-add-global-end-token end --delimiter '  ' --dict-tokenizer bytelevelbpe  --dropout 0.1 --fp16 True --init-model zoo:blender/reddit_3B/model --dict-file zoo:blender/reddit_3B/model.dict --label-truncate 128 --log_every_n_secs 10 -lr 7e-06 --lr-scheduler reduceonplateau --lr-scheduler-patience 3 --optimizer adam --relu-dropout 0.0 --activation gelu --model-parallel true --save-after-valid True --text-truncate 128 --truncate 128 --warmup_updates 100 --fp16-impl mem_efficient --update-freq 2 --gradient-clip 0.1 --skip-generation   False -vp 4 -vmt ppl -vmm min --model-file /py_torch/parlai_finetune_model/test_train_27B

**Comment by klshuster** [comment]
`episode_done: True` tells the model to stop accumulating context in its representation of the dialogue history. if you have individual, separate q&a pairs from a dataset, then you want each to be `True`.
